src/main/java/core/basesyntax/service/impl/WriterToCsvImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.WriterToCsv;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriterToCsvImpl implements WriterToCsv {
+    @Override
+    public void write(String data, String toFileName) {
```

Comment:
The method parameter 'toFileName' should not be hardcoded in the service implementation. It's better to accept file paths as parameters to make the service more flexible and testable.